Stratis.Bitcoin/FullNode.cs

```diff
@@ -245,6 +249,15 @@ private void StartPeriodicLog()
 						                     this.ChainBehaviorState.HighestPersistedBlock.HashBlock);
 					}
 
+					if (this.Wallet != null)
+					{
+						var height = this.Wallet.LastBlockHeight();
```

Comment:
if there are no wallets, this will show the height of the chain, and the log will lead to confusion.